x-pack/plugins/fleet/server/routes/agent/upgrade_handler.ts

```diff
@@ -67,6 +67,15 @@ export const postAgentUpgradeHandler: RequestHandler<
       }
     }
 
+    if (hasAgentBeenUpgradedRecently(agent)) {
+      return response.customError({
+        statusCode: 400,
+        body: {
+          message: `agent ${request.params.agentId} was upgraded less than 10 minutes ago`,
```

Comment:
This error message should probably interpolate the `UPGRADE_COOLDOWN_IN_MIN` constant so we don't forget to update it if that constant ever changes.

Reply:
Good catch! Thanks for reviewing 👍